### ISSUE TITLE
Fuel consumption should depend on the vehicle

### ISSUE THREAD

**Issue by atg-abhishek** [AndreaFodor/Elevate#2]
I would recommend linking this up with the Car class implementation for more accurate results as the linked asset in the comment above the ref indicates that this is for a specific vehicle type

**Ref:**
https://github.com/AndreaFodor/Elevate/blob/84962ccc10c37c695af54194877cf40710e17f18/fuelConsumption.py#L16

**Comment by AndreaFodor** [comment]
Yes, that is still in development, we didn't have time to fully implement that feature. The idea was to find a database with different car models and implement that.  Thanks for your comment!
